README.release.md

```diff
@@ -0,0 +1,49 @@
+# Creating a New Release
+
+## Changelog
+
+The changelog is autogenerated by github at the release notes by inspecting 
+labels at PRs.
+
+To have a proper changelog, make sure the PRs are labeled as expected so they 
+they appear at proper the proper section according to [1] this are the labels:
+
+- Skip release notes:
+  - ignore-for-release
+- Breaking Changes 🛠
+  - Semver-Major
+  - breaking-change
+- Exciting New Features 🎉
+  - Semver-Minor
+  - enhancement
+- Bug Fix 🐛
+  - Semver-Patch
+  - bug
+- Other changes
+  - Everything that has no labels at all
+
+[1] https://github.com/nmstate/nmpolicy/blob/main/.github/release.yml
+
+## Tagging
+
+- [ ] Tag new release in git.
+```bash
+# Make sure your local git repo is sync with upstream.
+# The whole version string should log like `v0.1.0`.
+# For the commit message use the following format: `nmpolicy 0.1.0 release`.
+git tag --sign v<version>
```

Comment:
How does this work with the labels defined above? What if I add a label "Sebver-Major breaking-change" and then try to post a tag with patch release?

Reply:
A patch release cannot contain a  breaking change.

Reply:
But what prevents me from adding that label and then tagging?

And what happens when I do that? Will the CI refuse to post a new release but the tag will stay on GitHub?

Reply:
The labels only affect the changelog, the release will be created, I don't understand what you mean.

Reply:
I wonder what happens in this scenario:

1. I post a PR that adds breaks API to main
2. I add the "breaking-change" and "Semver-Major" labels to the PR
3. The PR gets merged
4. A maintainer unaware of this PR wants to release a new minor version to get new changes to knmstate
5. The maintainer posts a tag with minor version bump

What happens then? How will the automation deal with a situation where the changelog has "breaking-change"  and "Semver-Major" but the posted tag is only a bump on minor version, hence breaking semver.

Reply:
The purpose of the automation is not to prevent that, like the other kubevirt projects, if you make a breaking change you should do a major releas and the like, there is no automation to prevent the issue.

Reply:
Ack, thanks for explaining